db/migrate/20190313203139_add_created_by_column.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedByColumn < ActiveRecord::Migration[5.2]
+  def change
+    add_column :foods, :created_by, :string
```

Comment:
Why is this a string? Seems like maybe it would make more sense for it to be an ID since you're using `has_many` above. `t.references` I think - like in this example. https://stackoverflow.com/questions/16309742/t-references-in-the-migration-vs-belongs-to-in-the-model